ENIGMAsystem/SHELL/Widget_Systems/xlib/Info/About.ey

```diff
@@ -10,3 +10,6 @@ Author: Samuel Venable
 
 Depends:
   Build-Platforms: MacOSX, Linux, FreeBSD
+
+Represents:
+  Build-Platforms: Linux, FreeBSD
```

Comment:
We should delay this until the vulnerability with `execute_program` is fixed. Otherwise arbitrary code execution (first-party or third-party) is as simple as showing a bad message.

Reply:
@JoshDreamland what do you even mean? we already prompt the user whether they want to allow it. What more are you considering a vulnerability? I consider it a vulnerability that fundies as toxic of a person he is has any say in the projects direction and if find it crazy you see anything eye to eye with this total creep.